web/models.py

```
@@ -1532,7 +1599,7 @@ def save(self, *args, **kwargs):
                     import logging
 
                     logger = logging.getLogger(__name__)
-                    logger.error(f"Error calculating streak for user {self.user.id}: {e}")
+                    logger.error(f"Error calculating streak for userId {self.user.id}: {e}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)